vllm/engine/metrics.py

```diff
@@ -15,72 +14,51 @@
 
 logger = init_logger(__name__)
 
-disable_created_metrics()
 
 # The begin-* and end* here are used by the documentation generator
 # to extract the metrics definitions.
 
-
 # begin-metrics-definitions
 class Metrics:
     labelname_finish_reason = "finished_reason"
 
-    def __init__(self, labelnames: List[str], max_model_len: int):
-        # Unregister any existing vLLM collectors
```

Comment:
This was a hack to enable our CI to pass before

`REGISTRY` is a global value. We were touching internal state since our CI runs `LLM` twice

I just updated the CI to run with `--forked` which obviate the need for this hack

Reply:
We had problems using --forked before due to cuda issue (and we removed it). https://github.com/vllm-project/vllm/pull/3631

Seems like metrics test also use the vllm instances, and I am a little concern it will introduce the similar issue again. Why don't we just keep this hack instead?

Reply:
As long as the test does not import torch, —forked is fine